x-pack/plugins/enterprise_search/public/applications/analytics/components/layout/page_template.tsx

```diff
@@ -27,26 +33,60 @@ interface EnterpriseSearchAnalyticsPageTemplateProps extends PageTemplateProps {
 export const EnterpriseSearchAnalyticsPageTemplate: React.FC<
   EnterpriseSearchAnalyticsPageTemplateProps
 > = ({ children, analyticsName, pageChrome, pageViewTelemetry, ...pageTemplateProps }) => {
+  const { updateSideNavDefinition, getChromeStyle$ } = useValues(KibanaLogic);
+  const chromeStyle = useObservable(getChromeStyle$(), 'classic');
+  const alwaysReturnNavItems = true;
+
+  const navItems = useEnterpriseSearchAnalyticsNav(
```

Comment:
We have to have different hooks for classic & project based Nav if we want to exclude Enterprise Search in the new nav.

Reply:
I don't think this is necessary. The new nav is driven by the navigation tree definition. If Enterprise Search is not there, it won't appear.